src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
One possible issue is that the whitelist is generated dynamically at build time based on what Kenton's machine detects as the set of files needed to perform DNS resolution on his system, which may differ from that of other systems.  For instance, I suspect he does not use resolvconf, which leads to its absence from `etc.list` in our published packages.

Perhaps we should add some explicit whitelist entries, and autodetect the rest.